core/overlays/ocp4-stage/common/configmaps.yaml

```diff
@@ -38,7 +38,6 @@ data:
   solvers: |
     solver-rhel-8-py36
     solver-rhel-8-py38
-    solver-fedora-32-py37
```

Comment:
it does not be decommissioned in cnv-prod?

Reply:
we dont run fedora solver in prod yet .

Reply:
We wanted to sync with s2i provided. We do not have f32-py37 https://github.com/thoth-station/s2i-thoth/